tutorials/fundamentals/merge.md

```diff
@@ -0,0 +1,22 @@
+To merge two objects in JavaScript, you can use the [spread `...` operator](/tutorials/fudnamentals/spread).
```

Comment:
Add "The spread operator creates a new object with all the properties from the first object and second object. If there's two properties with the same name, the property from the second object wins out."